adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParser.java

```diff
@@ -185,6 +185,28 @@ public ParsedEntity parse(RawEntity toParse) {
                                 ));
                     }
 
+                /*
+                } else if (columnSpecProto.getType().getType() ==
+                        GtfsSpecificationProto.ColumnInputType.InputType.TIME) {
+
+                    //Shouldn't we implement this elsewhere? Custom validator or class?
+                    Integer timeValueAsInt = null;
+                    if (rawField != null && rawField.matches("([0-9][0-9]|[0-9]):[0-5][0-9]:[0-5][0-9]")) {
```

Comment:
If this code stays, the regex in `.matches(...)` should be pulled out into a `Pattern` and `Matcher` which is re-used across rows. Regex can be expensive to compile and should be re-used when possible - see https://www.baeldung.com/java-regex-performance for details.

Reply:
It will be implemented in TimeConversionUtils class cc @lionel-nj

Reply:
See 4aee60541096bf230bead2d0b3d9b3d3938bc84e

Reply:
See 7577738eb1611d455514121fd20aebb3f5b31718